build.gradle

```diff
@@ -92,8 +92,22 @@ subprojects {
     tasks.check.dependsOn(tasks.jacocoTestReport)
     tasks.check.dependsOn(javadoc)
 
-    if (System.env.CIRCLE_TEST_REPORTS) {
-        test.reports.junitXml.destination = new File(System.env.CIRCLE_TEST_REPORTS, it.getName())
+    if (System.env.CIRCLECI) {
+        test.reports.junitXml.destination = new File(System.env.TESTS_DIR, it.getName())
+
+        task copyCiArtifactsToCircle {
```

Comment:
I guess I don't understand why we can't do what we were doing before: set a directory, write all the stuff there.